src/odemis/driver/xt_client.py

```diff
@@ -358,6 +358,16 @@ def check_and_transfer_latest_package(self) -> None:
                     update.show()
                 else:
                     logging.warning("{} is a bad file in {} not transferring latest package.".format(ret, package.path))
+
+                # This version supports reading raw stage coordinates
+                if current_version is not None and bitness is not None:
+                    if current_version <= "1.12.0":
```

Comment:
FYI, you cannot compare versions with string comparison. For instance "9" > "10"! There are dedicated functions for that.